src/functions/getProductsList/handler.ts

```diff
@@ -1,10 +1,41 @@
 import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
 import { formatJSONResponse } from '@libs/api-gateway';
 import { middyfy } from '@libs/lambda';
-import { products } from '../../mocks/products';
+import { DynamoDBClient, ScanCommand } from '@aws-sdk/client-dynamodb';
+import { unmarshall } from '@aws-sdk/util-dynamodb';
+
+const prettify = (data:any) => data.Items?.map((item) => unmarshall(item));
 
 const getProductsList: ValidatedEventAPIGatewayProxyEvent<any> = async (event) => {
-	return formatJSONResponse({ products, statusCode: 200 });
+
+	try {
+		const dbClient = new DynamoDBClient({ region: 'eu-west-1' });
+		const [productsData, stocksData] = await Promise.all([
+			dbClient.send(new ScanCommand({ TableName: process.env.PRODUCTS_TABLE_NAME })),
```

Comment:
FYI
More readable and simple syntax:
`await dbClient.scan({ TableName: process.env.YOUR_TABLE_NAME }`